AuthenticatorShared/Core/Platform/Services/NotificationCenterService.swift

```diff
@@ -0,0 +1,36 @@
+import Combine
+import UIKit
+
+// MARK: - NotificationCenterService
+
+/// A protocol for a `NotificationCenterService` which accesses the app's notification center.
+///
+protocol NotificationCenterService: AnyObject {
+    /// A publisher for when the app enters the background.
+    ///
+    func didEnterBackgroundPublisher() -> AnyPublisher<Void, Never>
```

Comment:
Should we remove this definition since it's not being used?

Reply:
I debated that as well. I felt like keeping it since they are kind of a pair and it might be useful in the future. But I'm also fine removing it since it's a fairly obvious thing to add when we do need it.

Reply:
I'd lean towards removing it because it's easy to add back, and may never be used.

Reply:
I'd vaguely lean towards keeping it to maintain parity with the PM app

Reply:
Good point. Works for me.